src/js/models/appUser.js

```diff
@@ -12,22 +13,36 @@ var AppUser = module.exports = Backbone.AssociatedModel.extend({
     },
 
     parse: function(data) {
-        return _.isObject(data) ? data : {
-            _id: data
-        };
+        return data.appUser;
     },
 
     url: function() {
-        return 'appusers/' + this.id;
+        return 'v1/appusers/' + this.id;
```

Comment:
is this not relative to the root you defined above:

`"API_URL": "https://api.supportkit.io/v1"`